doc/configuration.md

```diff
@@ -183,7 +183,7 @@ or directory from a dev dependency, you can do so by adding it via one of the fo
 binary files, the regular file will take precedence.
 
 
-### Force auto-discovery (`auto-discovery`)
+### Force auto-discovery (`force-autodiscovery`)
 
 The `auto-discovery` (`bool` default `false`) setting forces Box to attempt to find which files to include even though
```

Comment:
shall be renamed as well?

change of input name (CLI param, param in config file) is a BC breaker